ui/pages/confirmations/components/confirm/info/dapp-swap-comparison-utils.ts

```diff
@@ -0,0 +1,167 @@
+import { Hex } from '@metamask/utils';
+import { Interface, TransactionDescription } from '@ethersproject/abi';
+import { QuoteResponse } from '@metamask/bridge-controller';
+import { addHexPrefix } from 'ethereumjs-util';
+import { getNativeTokenAddress } from '@metamask/assets-controllers';
+
+import { Numeric } from '../../../../../../shared/modules/Numeric';
+
+export const ABI = [
+  {
+    constant: true,
+    inputs: [
+      {
+        name: 'commands',
+        type: 'bytes',
+      },
+      {
+        name: 'inputs',
+        type: 'bytes[]',
+      },
+      {
+        name: 'deadline',
+        type: 'uint256',
+      },
+    ],
+    name: 'execute',
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [
+      {
+        name: 'commands',
+        type: 'bytes',
+      },
+      {
+        name: 'inputs',
+        type: 'bytes[]',
+      },
+    ],
+    name: 'execute',
+    type: 'function',
+  },
+];
+
+const getWordsFromInput = (input: string) => {
```

Comment:
Are words not just 8 bytes on modern systems?

But we're defining them as 32 bytes / 64 hex characters?

Should we call them arguments instead?

Reply:
PR us updated to use `args` in place of `words`